src/current/v23.1/migrate-from-mysql.md

```diff
@@ -363,23 +363,16 @@ Use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to check
 
 1. [Install MOLT Verify.]({% link {{ page.version.version }}/molt-verify.md %})
 
-1. In the directory where you installed MOLT Verify, use the following command to compare the two databases, specifying the [JDBC connection string for MySQL](https://dev.mysql.com/doc/connector-j/8.1/en/connector-j-reference-jdbc-url-format.html) {% comment %}with `--source`{% endcomment %} and the SQL connection string for CockroachDB{% comment %}with `--target`{% endcomment %}:
+1. In the directory where you installed MOLT Verify, use the following command to compare the two databases, specifying the [JDBC connection string for MySQL](https://dev.mysql.com/doc/connector-j/8.1/en/connector-j-reference-jdbc-url-format.html) with `--source` and the SQL connection string for CockroachDB with `--target`:
```

Comment:
i learnt yesterday this isn't actually a jdbc string, but whatever "DSN" our driver accepts - format from https://github.com/go-sql-driver/mysql#dsn-data-source-name. Sorry for the mislead.

We'll make it an actual JDBC string soon (jeremy's working on it) - my README is currently wrong.

Reply:
I see. However, should the `jdbc` prefix remain in the MySQL string for now? It worked for me when testing these steps.

Reply:
i'd say remove it (though it still works)